notebooks/basics/quantum-circuits.ipynb

```diff
@@ -1298,17 +1303,17 @@
     "$$\n",
     "\n",
     "The vectors $\\vert \\psi \\rangle$ and $\\vert \\phi \\rangle$ are then said to *differ by a global phase*.\n",
-    "We also sometimes refer to $\\alpha$ as being or representing a *global phase*, although this is context-dependent:\n",
-    "any number on the unit circle can be thought of as a global phase when it is multiplied to some unit vector.\n",
+    "We also sometimes refer to $\\alpha$ as a *global phase*, although this is context-dependent:\n",
+    "any number on the unit circle can be thought of as a global phase when multiplied by a unit vector.\n",
```

Comment:
"multiplied to" rather than "multiplied by"

(Intuitively I feel like we multiply vectors by scalars rather than multiplying scalars by vectors, even though both make sense technically.)